index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Mini Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <div class="logo">
+      <img src="images/Spartalogo.png" width="200" alt="">
+    </div>
+    <div class="links">
+      <nav>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+    </div>
+    <div class="banner">
+      <img src="images/redbanner.jpg" width="100%" height="150px" alt="">
+    </div>
+  </head>
+
+  <body>
+    <div class="container">
+      <div class="left">
```

Comment:
I would maybe put some spaces when you make one big div with so much content in it as you can more clearly see the structure of the site at a glance, but that's just preference - nothing wrong with this indentation.